packetbeat/beater/packetbeat.go

```diff
@@ -79,8 +57,8 @@ func init() {
 	}
 }
 
-func New(b *beat.Beat, rawConfig *common.Config) (beat.Beater, error) {
-	config := config.Config{
+func initialConfig() config.Config {
```

Comment:
Could the changes on packetbeat move to its own independent PR to decouple it from adding it to the agent?

Reply:
@ruflin : done #22145